aspnetcore/signalr/hubcontext.md

```diff
@@ -54,6 +54,13 @@ app.Use(async (context, next) =>
 
 > [!NOTE]
 > When client methods are called from outside of the `Hub` class, there's no caller associated with the invocation. Therefore, there's no access to the `ConnectionId`, `Caller`, and `Others` properties.
+>
+> If your application needs to map a user to the connection ID and persist that mapping, you can do one of the following:
+>
+> - Persist mapping of single or multiple connections as groups. See [Groups in SignalR](xref:signalr/groups#groups-in-signalr) for more information.
+> - Retain connection and user information in a singleton service. See [Inject services into a hub](xref:signalr/hubs#inject-services-into-a-hub) for more information.
+> - Retain connection and user information in a dictionary that is stored in memory.
```

Comment:
It's not just in-memory, you can store the info in a database or any other storage method.